wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -724,28 +711,55 @@ private JsonObject getHuggingFaceModelConfig() throws ModelException, IOExceptio
         try (InputStream is = modelConfigUri.toURL().openStream();
                 BufferedReader reader =
                         new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8))) {
-            return JsonUtils.GSON.fromJson(reader, JsonElement.class).getAsJsonObject();
-        } catch (IOException e) {
-            logger.error(
-                    "The huggingface hub model_id {} is not valid. Please verify this is a valid"
-                            + " model_id",
-                    modelId);
-            throw e;
+            Gson gson =
+                    JsonUtils.builder()
+                            .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
```

Comment:
Why we need this since we already use @SerializedName?

Reply:
left over from my testing - this should be removed good catch